src/Test.UnitTests.Sarif.Driver/Sdk/PluginDriverCommandTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+using FluentAssertions;
+
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.CodeAnalysis.Sarif.Driver;
+
+using Moq;
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Test.UnitTests.Sarif.Driver.Sdk
+{
+    public class PluginDriverCommandTests
+    {
+        [Fact]
+        public void PluginDriverCommand_ValidateInvocationPropertiesToLog_ShouldValidateParameters()
```

Comment:
>PluginDriverCommand_ValidateInvocationPropertiesToLog_ShouldValidateParameters [](http://example.com/codeflow?start=20&length=78)

Here's another bad use of the test cases pattern. 

Please don't do this for simple test scenarios.

Reply:
refactored.